enclave/core/sgx/internal.c

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <openenclave/internal/print.h>
+#include "internal_t.h"
+
+int oe_internal_ping_ecall(int value)
```

Comment:
Similarly I think this file has no SGX-specific contents and so I don't understand why it's under the core/sgx directory

Reply:
This file is a placeholder for functions that currently reside in the sgx directory.

Reply:
The `ping` function is a placeholder. I think Mike will replace it with internal sgx-specific ECALLs.

Reply:
Yes, that's correct: the ping function is just a placeholder because EDL does not allow empty function tables. They will be removed and SGX-specific calls will be added. This infrastructure is part of the internal SGX work.